### ISSUE TITLE
Alt text is not descriptive [.../hosting files]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7066]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/guides/hosting/git-based-deployments.mdx:
```
  44  
  45: ![Select Get Started](/images/hosting/git/3.png)
  46  

  48  
  49: ![Select GitHub](/images/hosting/git/4.png)
  50  

  52  
  53: ![Authenticate](/images/hosting/git/5.png)
  54  

  56  
  57: ![Build settings](/images/hosting/git/6.png)
  58  

  60  
  61: ![Review details](/images/hosting/git/7.png)
  62  

  64  
  65: ![Successful deployment](/images/hosting/git/8.png)
  66  

  68  
  69: ![View your app](/images/hosting/git/9.png)
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/guides/hosting/nextjs.mdx:
```
  131  
  132: ![CLI Output](/images/hosting/next/cli-output.png)
  133  

  183  
  184: ![Rewrites](/images/hosting/next/rewrites.png)
  185  

  272  
  273: ![image](/images/start-nextjs-deploy-1.png)
  274  

  276  
  277: ![image](/images/start-nextjs-deploy-2.png)
  278  

  284  
  285: ![image](/images/start-nextjs-deploy-3.png)
  286  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/pages/[platform]/start/getting-started/hosting/index.mdx:
```
  101  
  102: ![image](/images/browser-vanillajs-hosting.png)
  103  

  107  
  108: ![image](/images/browser-published-updated.png)
  109  
```
